ledger/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const MAINNET_BROADCAST_URI = 'https://blockstream.info/api/tx';
```

Comment:
We should use the API URL selected by the user for transaction broadcast instead of the hardcoded value here.

https://github.com/secretkeylabs/xverse-core/blob/develop/api/esplora/esploraAPiProvider.ts#L153

Reply:
Could I get more context on where the user selects it?

Reply:
https://github.com/secretkeylabs/xverse-core/blob/develop/api/esplora/esploraAPiProvider.ts#L153